content/docs/command-reference/add.md

```diff
@@ -336,91 +339,78 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Caching large data externally {#straight-to-cache}
```

Comment:
hmm ... the name is still very confusing to me. have discussed something like `Add large data from an external location`?

Reply:
> Add large data from an external location

The problem with that is that it applies also to straight-to-remote (next example). Need to think... 🧠

Reply:
OK, let's go with "Large data from an external location" here since the cache is the base case, and figure something out for the straight-to-remote example title.

Reply:
Moved discussion to https://github.com/iterative/dvc.org/pull/2302#issuecomment-801442953, resolving here.